quickstarts/lighttpd/config.yml

```diff
@@ -0,0 +1,40 @@
+
+slug: lighttpd
+description: |
+  ## Comprehensive monitoring quickstart for Lighttpd
+
+  Lighttpd is an open-source web server designed for high-performance situations while staying standards-compliant, secure and adaptable.
+
+  ### Why monitor Lighttpd?
+
+  Lighttpd monitoring provides important information to help the developers construct a complete picture of a web server's performance. This data comprises uptime, network in bytes and packets, number of connections and more.
+
+  ### What’s included?
+
+  The Lighttpd monitoring quickstart from New Relic provides immediate full-stack observability:
+  - Alerts (average number of connections, average number of requests, high memory usage).
+  - Dashboards (uptime, connection state handle request, connection state still alive, networks, requests per second and more).
+  - Be proactive in monitoring potential issues and resolving them as soon as possible.
```

Comment:
I'd delete this last bullet. It doesn't seem very helpful or informative.

If we must keep it, I'd rewrite it to:

"Deeper insight into your server's performance issues."